configDesktop.js

```diff
@@ -9,223 +9,217 @@ const {
-		path: '/w/index.php?title=Test&action=history'
-	},
-	{
-		label: 'Talk:Test (#vector-2022)',
-		path: '/wiki/Talk:Test'
+		label: 'TOC pinned with hidden subsections (#vector-2022, #toggle-toc-subsections)',
+		path: '/wiki/Test'
 	},
 	{
-		label: 'Tree (#vector-2022)',
-		path: '/wiki/Tree'
+		label: 'TOC unpinned in page title (#vector-2022, #toc-unpinned, #toc-open)',
+		path: '/wiki/Test'
 	},
 	{
-		label: 'Test (#vector-2022, #logged-in)',
-		path: '/wiki/Test',
-		selectors: [ 'html' ]
+		label: 'TOC unpinned below page title (#vector-2022, #scroll, #toc-unpinned, #toc-open)',
+		path: '/wiki/Test'
 	},
 	{
-		label: 'Test (#vector-2022, #logged-in, #limited-width-disabled)',
+		label: 'TOC pinned below page title (#vector-2022, #scroll)',
 		path: '/wiki/Test',
 		viewports: [
+			VIEWPORT_DESKTOP,
+			VIEWPORT_DESKTOP_WIDE,
 			VIEWPORT_DESKTOP_WIDEST
 		]
 	},
+	//
+	// Sticky header
+	//
 	{
-		label: 'Test (#vector-2022, #logged-in, #toc-unpinned, #main-menu-unpinned, #limited-width-disabled)',
+		label: 'Sticky header with pinned TOC (#vector-2022, #logged-in, #scroll)',
 		path: '/wiki/Test',
 		viewports: [
+			VIEWPORT_DESKTOP,
+			VIEWPORT_DESKTOP_WIDE,
 			VIEWPORT_DESKTOP_WIDEST
 		]
 	},
 	{
-		label: 'Test (#vector-2022, #logged-in, #toc-unpinned, #main-menu-unpinned, #page-tools-unpinned, #limited-width-disabled)',
+		label: 'Sticky header with unpinned TOC (#vector-2022, #logged-in, #scroll, #toc-unpinned, #toc-open)',
 		path: '/wiki/Test',
 		viewports: [
+			VIEWPORT_DESKTOP,
+			VIEWPORT_DESKTOP_WIDE,
 			VIEWPORT_DESKTOP_WIDEST
 		]
 	},
 	{
-		label: 'Test (#logged-in, #vector-2022, #scroll, #search-sticky, #search-focus)',
+		label: 'Sticky header full width (#vector-2022, #logged-in, #scroll, #limited-width-disabled)',
 		path: '/wiki/Test',
-		// No search icon present if no sticky header present, so limit to this viewport
 		viewports: [
-			VIEWPORT_DESKTOP_WIDE
+			VIEWPORT_DESKTOP,
+			VIEWPORT_DESKTOP_WIDE,
+			VIEWPORT_DESKTOP_WIDEST
 		]
 	},
+	//
+	// Search
+	//
 	{
-		label: 'Special:BlankPage with user menu open (#vector-2022, #logged-in, #user-links-open)',
-		path: '/wiki/Special:BlankPage'
+		label: 'Search in header (#vector-2022, #search-focus)',
+		path: '/wiki/Test'
 	},
 	{
-		label: 'Test sticky header (#vector-2022, #logged-in, #scroll)',
+		label: 'Search loading in header (#vector-2022, #search-focus, #search-offline)',
+		path: '/wiki/Test'
+	},
+	{
+		label: 'Search in sticky header (#logged-in, #vector-2022, #scroll, #search-sticky, #search-focus)',
 		path: '/wiki/Test',
 		viewports: [
-			VIEWPORT_DESKTOP
+			VIEWPORT_DESKTOP,
+			VIEWPORT_DESKTOP_WIDE,
+			VIEWPORT_DESKTOP_WIDEST
 		]
 	},
+	//
+	// Dropdowns
+	//
 	{
-		label: 'Test sticky header (#vector-2022, #limited-width-disabled, #logged-in, #scroll)',
+		label: 'Anon user links dropdown (#vector-2022, #user-links-open)',
 		path: '/wiki/Test',
+		selectors: [ '#vector-user-links-dropdown .vector-dropdown-content' ],
```

Comment:
I wish there was a way to capture both the button  _and_ the menu together, as a regression could affect the position of the menu  (e.g. move it near the footer of the page), and I think this would still pass, since it only captures the menu selector. I tried to add the button selector to the array, but this didn't capture them together.

How strongly do you feel about limiting this to this selector rather than capturing the viewport?

Reply:
yeah, originally i had the button and the menu in the selectors here, but pixel doesnt put them together, it just shows up as two separate cases. 

I dont mind removing the selector, i guess it should still capture any changes the same.